packages/airnode-node/src/evm/fulfillments/api-calls.test.ts

```diff
@@ -19,7 +19,7 @@ import * as fixtures from '../../../test/fixtures';
 import * as wallet from '../wallet';
 import { GasTarget, RequestErrorMessage, RequestStatus } from '../../types';
 import { AirnodeRrp } from '../contracts';
-import { MAXIMUM_ONCHAIN_ERROR_LENGTH } from '../../constants';
+import { MAXIMUM_ONCHAIN_ERROR_LENGTH, GAS_LIMIT } from '../../constants';
```

Comment:
Why importing it in this test and not the others (e.g. process-transactions.test)

Reply:
That was fast! I was actually going to ask if we should use the same constant in every place with the same `gasLimit` value, or only in the places where it had been defined as a constant previously.

Reply:
Feel free to use it as variable everywhere.